pkgs/development/python-modules/llama-cpp-python/default.nix

```diff
@@ -99,7 +105,9 @@ buildPythonPackage rec {
 
   passthru = {
     updateScript = gitUpdater { rev-prefix = "v"; };
-    tests.llama-cpp-python = llama-cpp-python.override { cudaSupport = true; };
+    tests.llama-cpp-python = llama-cpp-python.override {
```

Comment:
the purpose of `tests.llama-cpp-python` was locally test cuda build. ofborg and hyrda don't build unfree software

so if we apply this change, it will no do anything other that just build llama-cpp-python

Reply:
There's no CUDA for Darwin, so what should we do here? Set passthru.tests only when it's not Darwin, otherwise leave unset?

Reply:
it's common and ok to leave as it is, it's just relies on knowledge that unfree software is skipped by ofborg and therefore `tests.llama-cpp-python` will look like passing on ofborg

or as you suggested, only set `tests.llama-cpp-python` for linux platforms